doc/rst/tools/mason/mason.rst

```diff
@@ -904,6 +904,7 @@ Create a local registry:
    git add README.md /Bricks/DummyPackage/0.1.0.toml
    git commit -m 'First Commit'
 
+Alternately, you may automatically create a local registry by running ``mason publish -c|--create-registry path/to/local/registry``.
```

Comment:
I suggest we keep this example simple and stick to one of the flags here, maybe `--create-registry` since it's more explicit.